### PR TITLE
ICDS Dashboard: Adding awc site code to tabular reports

### PR DESCRIPTION
**Pull request by saiprakash-v**
https://app.asana.com/0/1172925702295169/1143121160741098
##### SUMMARY

This PR adds awc site code column in Children, Pregnant women, Demographics, System usage and AWC Infrastructure, THR tabular reports.

[THR Report__days_beneficiary_wise__07-07-2020__20-45-56.xlsx](https://github.com/dimagi/commcare-hq/files/4885667/THR.Report__days_beneficiary_wise__07-07-2020__20-45-56.xlsx)
[THR Report__beneficiary_wise__07-07-2020__20-44-49.xlsx](https://github.com/dimagi/commcare-hq/files/4885668/THR.Report__beneficiary_wise__07-07-2020__20-44-49.xlsx)
[THR Report__consolidated__07-07-2020__20-43-25.xlsx](https://github.com/dimagi/commcare-hq/files/4885669/THR.Report__consolidated__07-07-2020__20-43-25.xlsx)
[AWC_Infrastructure (2).xlsx](https://github.com/dimagi/commcare-hq/files/4885670/AWC_Infrastructure.2.xlsx)
[System_Usage (5).xlsx](https://github.com/dimagi/commcare-hq/files/4885671/System_Usage.5.xlsx)
[Demographics (5).xlsx](https://github.com/dimagi/commcare-hq/files/4885672/Demographics.5.xlsx)
[Pregnant_Women (2).xlsx](https://github.com/dimagi/commcare-hq/files/4885674/Pregnant_Women.2.xlsx)
[ChildrenTabReport.xlsx](https://github.com/dimagi/commcare-hq/files/4885675/ChildrenTabReport.xlsx)

### CODE REVIEW

File: custom/icds_reports/migrations/0187_update_system_usage_report_view.py
Comment (Rohit25negi) [inline custom/icds_reports/migrations/0187_update_system_usage_report_view.py:1]: Could you check if the migration is on the right track. As in it has not conflict with master?
Also we actually name the new migration we add incrementing the count in the file name

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
Hey, Left a few comments about migration. But generally looks good apart from one point. Where does it tell the list of reports to have this change? I can't find it on ticket and description does not says  incentive report is being changed but it is changed.

File: custom/icds_reports/migrations/0187_update_system_usage_report_view.py
Comment (saiprakash-v) [reply custom/icds_reports/migrations/0187_update_system_usage_report_view.py:1]: I just looked up all the tabular reports and added awc site code in all the reports which has awc name column.
(I added incentive report later. So, I missed adding it in the description)
(Also regarding the migration files, I will update it as soon as this pr: https://github.com/dimagi/commcare-hq/pull/27201 is approved)

**Comment by saiprakash-v** [comment]
@ShubhamBansal1997 Please review the changes

**Review by ShubhamBansal1997 — CHANGES_REQUESTED** [review]
Please check tests failure, they look related. Also attach the excel files with data (just to ensure the reports are working fine.)

**Review by ShubhamBansal1997 — APPROVED** [review]
Overall Looks good, but wait for @Rohit25negi approval too.

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
Everything looks good except the Incentive report

**Comment by Rohit25negi** [comment]
My last review still looks unaddressed. @saiprakash-v

**Comment by saiprakash-v** [comment]
@Rohit25negi Change in AWW performance report is made as a seperate task here:  https://app.asana.com/0/1185205808600689/1184316228725185

**Pull request closed without merging**
